src/test/java/dk/cngroup/lentils/service/ScoreServiceTest.java

```diff
@@ -0,0 +1,120 @@
+package dk.cngroup.lentils.service;
+
+import dk.cngroup.lentils.LentilsApplication;
+import dk.cngroup.lentils.entity.*;
+import dk.cngroup.lentils.repository.CypherRepository;
+import dk.cngroup.lentils.repository.HintTakenRepository;
+import dk.cngroup.lentils.repository.StatusRepository;
+import dk.cngroup.lentils.repository.TeamRepository;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+import static org.junit.Assert.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = {LentilsApplication.class, ObjectGenerator.class})
+public class ScoreServiceTest {
+
+    @InjectMocks
+    ScoreService scoreService;
+
+    @Autowired
+    private ObjectGenerator generator;
```

Comment:
Tohle prosím napiš jako private ObjectGenerator generator = new ObjectGenerator();
a odstraň ty anotace třídy @RunWith a @SprintBootTest.

V některých třídách to ještě je, v některých už se to opravovalo. Jde o to, že se pak nebude inicializovat zbytečně Spring jen kvůli ObjectGeneratoru a testy tak poběží rychleji.

Reply:
hotovo